### PR TITLE
Merge improvements from master into Custom_Keys

### PR DESCRIPTION
**Pull request by Reevak05**
There have been various improvements to `master` and this seeks to add them to `Custom_Keys_Developement_Branch`.

P.S. Development is spelled wrong

### CODE REVIEW

**Comment by MetalTurtle18** [comment]
Why did you add while 0 == 0

File: Affine Cipher.py [hunk 1]
```diff
@@ -1,38 +1,53 @@
 # v0.9
+# infinite loop
+while 0 == 0:
+    result = ""
```
Comment (MetalTurtle18) [inline Affine Cipher.py:4]: what is this for?

File: Affine Cipher.py [hunk 2]
```diff
@@ -1,38 +1,53 @@
...
-select = input("Type E for encryption and D for decryption")
-code_input = input("Enter the Word")
+    # input
+    select = input("Enter E for encryption or D for decryption: ")
+    if select == "E":
+        code_input = input("Enter word to be encoded: ")
+    elif select == "D":
+        code_input = input("Enter word to be decoded: ")
+    else:
+        code_input = "error"
```
Comment (MetalTurtle18) [inline Affine Cipher.py:22]: just do 
`if select == "E" or select == "D"`

File: Affine Cipher.py (hunk 1 above)
Comment (Reevak05) [reply Affine Cipher.py:4]: It makes the program run again after the encoding is finished. It just ends up being a nicer experience.

File: Affine Cipher.py (hunk 2 above)
Comment (Reevak05) [reply Affine Cipher.py:22]: No. The whole point of me changing that was so it says "encoded" or "decoded", depending on which one the user selects.

File: Affine Cipher.py (hunk 1 above)
Comment (MetalTurtle18) [reply Affine Cipher.py:4]: Doesn't is constantly set `result` to ` `? So how can you set `result` to anything else to give output?

File: Affine Cipher.py (hunk 2 above)
Comment (MetalTurtle18) [reply Affine Cipher.py:22]: Oh I see. I didn't see that before.

File: Affine Cipher.py [hunk 3]
```diff
@@ -1,38 +1,53 @@
...
-            number_input = int(code_input)
-        else:
-            number_input = (ord(str(my_list[i]).lower()) - 97)
-            print(ord(str(my_list[i]).lower()))
-            print(number_input)
-        intresult = modInverse(number_input)
-        print("Decoded Number is " + str(intresult))
-        result = result + chr(intresult + 97).upper()
-    print("Decoded Word is " + result)
+    if code_input != "error":
```
Comment (MetalTurtle18) [inline Affine Cipher.py:24]: I understand what you are doing here but why don't you just restart as soon as you receive a non `E` or `D` input? This is complicating the program more.

File: Affine Cipher.py [hunk 4]
```diff
@@ -1,38 +1,53 @@
 # v0.9
```
Comment (MetalTurtle18) [inline Affine Cipher.py:1]: This should be different shouldn't it?

**Pull request merged**